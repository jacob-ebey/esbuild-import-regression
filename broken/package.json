{
  "name": "expected",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "node ./build.js && node ./dist/index.js"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "esbuild": "0.14.21"
  },
  "dependencies": {
    "uuid": "8.3.2"
  }
}
