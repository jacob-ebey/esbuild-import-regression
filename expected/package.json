{
  "name": "expected",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "node ./build.js && node ./dist/index.js"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "esbuild": "0.13.14"
  },
  "dependencies": {
    "uuid": "8.3.2"
  }
}
